server/src/main/webapp/WEB-INF/rails/webpack/views/pages/materials/modal.tsx

```diff
@@ -241,3 +241,54 @@ export class ShowUsagesModal extends Modal {
     </div>;
   }
 }
+
+interface EllipseAttrs {
+  text: string;
+}
+
+interface EllipseState {
+  expanded: Stream<boolean>;
+  setExpandedTo: (state: boolean, e: MouseEvent) => void;
+}
+
+class EllipseText extends MithrilComponent<EllipseAttrs, EllipseState> {
+  private static MIN_CHAR_COUNT = 80;
+
+  oninit(vnode: m.Vnode<EllipseAttrs, EllipseState>): any {
+    vnode.state.expanded = Stream();
+    vnode.state.expanded(false);
```

Comment:
IIRC, this is done because of typescript compilation error.
try this instead - `vnode.state.expanded = Stream<boolean>(false)`?

Reply:
I'll add this change in the next PR change. Thanks!